### DISCUSSION TITLE
Disease Ontology alternate IDs not supported by prepare_C2M2_submission.py

### DISCUSSION THREAD

**Discussion by jeremyjyang** [nih-cfde/published-documentation#332] [Q&A] [answered]
This is just an observation (not necessarily Q&A) intended to be of use to other submitters. I notice that alternate ID DOID:1173 is not allowed by required script prepare_C2M2_submission.py, with error: `FATAL: Term 'DOID:1173' has no name: aborting, please check integrity of 'disease' reference file(s).` The preferred ID is DOID:0050770, for "polycystic liver disease". Ref: https://disease-ontology.org/term/DOID:0050770/obo/.

**Comment by abradyIGS** [comment]
I will be able look into this in detail over the weekend -- sorry for delayed reply, stay tuned

**Comment by abradyIGS** [comment, marked as answer]
We can't accept alternate IDs as primary IDs in submissions; submitters must use the latter.

The prep script doesn't load alt_ids into C2M2 synonyms lists, however, and there's certainly no problem with doing that, so I'll add that functionality to the next script release (3/31) in order to support term search by alt ID.

**Comment by abradyIGS** [comment]
Update: alt IDs are now included as synonyms.
